View Modes

BarthelemyMahe opened the pull request:
View Modes

BarthelemyMahe committed:
View Modes

TheoDesbonnets commented on UMI3D-SDK/Assets/UMI3D SDK/ClientDevelopmentKit/Core/Runtime/UMI3DTransactionDispatcher.cs:
Je pense pas que ça soit voulu

BarthelemyMahe replied:
J'ai supprimé le log et fix ce qui posait problème

TheoDesbonnets commented on UMI3D-SDK/Assets/UMI3D SDK/EnvironmentDevelopmentKit/InteractionSystem/Runtime/UMI3DBrowserRequestDispatcher.cs:
```diff
@@ -119,6 +119,7 @@ public static void DispatchBrowserRequest(UMI3DUser user, uint operationKey, Byt
                         boneType = UMI3DSerializer.Read<uint>(container);
                         bonePosition = UMI3DSerializer.Read<Vector3Dto>(container);
                         boneRotation = UMI3DSerializer.Read<Vector4Dto>(container);
+                        UnityEngine.Debug.Log("Event AbstractInteraction " + UMI3DEnvironment.GetEntityInstance<AbstractInteraction>(interactionId));
```
Peux tu supprimer ce log ?

BarthelemyMahe committed:
Fix

Indeed there was problems
- the UMI3DTransactionDispatcher was missing
- UMI3DSerializerVectorModules was duplicated
- the UMI3DUser was not updated
- .meta weren't generated
- removed logs

BarthelemyMahe committed:
Remove log
